versions/centos_7_2.10.1

```diff
@@ -0,0 +1 @@
+2
```

Comment:
These files should have exact versions like `2.10.1-1-<commit-sha>`

Reply:
- When releasing 2.10.1 we should also update files *.2.10 and *.2
- If files *.2.10.0 are missing, we should add them too.

Reply:
For example, see https://github.com/tarantool/docker/commit/d9de887713ccdd89d6aeb9b94dd83c75d10b572c